src/cascadia/TerminalSettingsEditor/ObjectModel/ColorScheme.cpp

```diff
@@ -0,0 +1,825 @@
+            m_propertyChanged(*this, PropertyChangedEventArgs{ L"BrightBlueHexValue" });
+            m_propertyChanged(*this, PropertyChangedEventArgs{ L"BrightBlue" });
+        }
+    }
+
+    Brush ColorScheme::BrightBlueBrush()
+    {
+        return ColorScheme::colorToBrush(m_brightblue);
+    }
+
+    winrt::hstring ColorScheme::BrightBlueHexValue()
+    {
+        return colorToHex(m_brightblue);
+    }
+
+    void ColorScheme::BrightBlueHexValue(winrt::hstring hex)
+    {
+        // TODO: Dirty dirty hack - We should try to parse this string a little harder than checking that its 9 long
+        if (hex.size() == 9)
+        {
+            ColorScheme::BrightBlue(ColorScheme::hexToColor(hex));
+        }
+    }
+
+    // Cyan Handlers
+    Color ColorScheme::Cyan()
+    {
+        return m_cyan;
+    }
+
+    void ColorScheme::Cyan(Brush const& brush)
+    {
+        ColorScheme::Cyan(brushToColor(brush));
+    }
+
+    void ColorScheme::Cyan(Color const& color)
+    {
+        if (cmpColor(color, m_cyan))
+        {
+            m_cyan = color;
+            m_propertyChanged(*this, PropertyChangedEventArgs{ L"CyanBrush" });
+            m_propertyChanged(*this, PropertyChangedEventArgs{ L"CyanHexValue" });
+            m_propertyChanged(*this, PropertyChangedEventArgs{ L"Cyan" });
+        }
+    }
+
+    Brush ColorScheme::CyanBrush()
+    {
+        return ColorScheme::colorToBrush(m_cyan);
+    }
+
+    winrt::hstring ColorScheme::CyanHexValue()
+    {
+        return colorToHex(m_cyan);
+    }
+
+    void ColorScheme::CyanHexValue(winrt::hstring hex)
+    {
+        // TODO: Dirty dirty hack - We should try to parse this string a little harder than checking that its 9 long
+        if (hex.size() == 9)
+        {
+            ColorScheme::Cyan(ColorScheme::hexToColor(hex));
+        }
+    }
+
+    // BrightCyan Handlers
+    Color ColorScheme::BrightCyan()
+    {
+        return m_brightcyan;
+    }
+
+    void ColorScheme::BrightCyan(Brush const& brush)
+    {
+        ColorScheme::BrightCyan(brushToColor(brush));
+    }
+
+    void ColorScheme::BrightCyan(Color const& color)
+    {
+        if (cmpColor(color, m_brightcyan))
+        {
+            m_brightcyan = color;
+            m_propertyChanged(*this, PropertyChangedEventArgs{ L"BrightCyanBrush" });
+            m_propertyChanged(*this, PropertyChangedEventArgs{ L"BrightCyanHexValue" });
+            m_propertyChanged(*this, PropertyChangedEventArgs{ L"BrightCyan" });
+        }
+    }
+
+    Brush ColorScheme::BrightCyanBrush()
+    {
+        return ColorScheme::colorToBrush(m_brightcyan);
+    }
+
+    winrt::hstring ColorScheme::BrightCyanHexValue()
+    {
+        return colorToHex(m_brightcyan);
+    }
+
+    void ColorScheme::BrightCyanHexValue(winrt::hstring hex)
+    {
+        // TODO: Dirty dirty hack - We should try to parse this string a little harder than checking that its 9 long

```

Comment:
I mean at that point, you should also make it a helper instead of copypasta down the whole file.